src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -402,7 +402,7 @@ internal static (TF_DataType[] tfOutputTypes, DataViewType[] outputTypes, (Opera
                 // i.e. the first dimension (if unknown) is assumed to be batch dimension.
                 // If there are other dimension that are unknown the transformer will return a variable length vector.
                 // This is the work around in absence of reshape transformer.
-                var idims = shape.dims;
+                var idims = shape.dims?.Select(x => checked((int)x))?.ToArray();
```

Comment:
>? [](http://example.com/codeflow?start=38&length=1)

does the dims can be null at all? looks the code later assume never be null.

Reply:
Sure I can remove the null checks.
```suggestion
                var idims = shape.dims.Select(x => checked((int)x)).ToArray();
```